### PR TITLE
Updated documentation to reflect change in name from Hydra to Samvera …

### PR DESCRIPTION
**Pull request by brianmcbride**
… and updated README.md to include information about Connect 2019.

### CODE REVIEW

**Review by adamjarling — APPROVED** [review]
Looks good.  Thanks for the updates @brianmcbride

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 # How to Contribute
 
-We want your help to make Project Hydra great.
+We want your help to make Samvera great..
```
Comment (hackartisan) [inline CONTRIBUTING.md:3]: extra period

**Review by hackartisan — APPROVED** [review]
Thanks Brian

**Comment by hackartisan** [comment]
@richardGreen Can you please add a comment explaining why you closed this without merging?

**Pull request merged**